documentation/src/main/asciidoc/mapstruct-reference-guide.asciidoc

```diff
@@ -2124,7 +2136,7 @@ First calling a mapping method on the source property is not protected by a null
 
 The option `nullValueCheckStrategy = NullValueCheckStrategy.ALWAYS` will always include a null check when source is non primitive, unless a source presence checker is defined on the source bean.
 
-The strategy works in a hierarchical fashion. `@Mapper#nullValueCheckStrategy` will override `@MappingConfig#nullValueCheckStrategy`.
+The strategy works in a hierarchical fashion. `@Mapping#nullValueCheckStrategy` will override `@BeanMapping#nullValueCheckStrategy`, `@BeanMapping#nullValueCheckStrategy` will override `@Mapper#nullValueCheckStrategy` and `@Mapper#nullValueCheckStrategy` will override `@MappingConfig#nullValueCheckStrategy`.
```

Comment:
This is no longer the case right?

Reply:
?  this is about `nullValueCheckStrategy`.. not about `nullValueMappingStrategy` :smile:.

Reply:
So this is how it is currently, just the documentation was not correct then?

Reply:
yep. I forgot to update it after my last PR I think..